dapp/src/utils/sentry.js

```diff
@@ -14,7 +14,7 @@ export const initSentry = () => {
   const distDir = `${config.serverRuntimeConfig.rootDir}/.next`
   Sentry.init({
     enabled: isProduction || isStaging,
-    environment: isProduction ? 'production' : 'staging',
```

Comment:
I would leave this line as it was before and change the line 6 to (and add a comment): 
```
// in staging environment NODE_NEV is also set to 'production'
const isProduction = process.env.NODE_ENV === 'production' &&  process.env.STAGING !== 'true'
```